src/ui/public/url/__tests__/prepend_path.js

```diff
@@ -0,0 +1,57 @@
+import expect from 'expect.js';
+
+import { prependPath } from '../prepend_path';
+
+describe('url prependPath', function () {
+  describe('returns the relative path unchanged', function () {
+    it('if it is null', function () {
+      expect(prependPath(null, 'kittens')).to.be(null);
+    });
+
+    it('if it is undefined', function () {
+      expect(prependPath(undefined, 'kittens')).to.be(undefined);
+    });
+
+    it('if it is an absolute url', function () {
+      expect(prependPath('http://www.hithere.com/howareyou', 'welcome')).to.be('http://www.hithere.com/howareyou');
```

Comment:
This is just how it worked before, though if it was accidental or intentional, I'm not sure.